cicd/gitops/configuring-a-cluster-with-red-hat-openshift-gitops.adoc

```diff
@@ -0,0 +1,24 @@
+[id="configuring-a-cluster-with-red-hat-openshift-gitops"]
+= Configuring a cluster with Red Hat OpenShift GitOps
+include::modules/common-attributes.adoc[]
+:context: configuring-a-cluster-with-red-hat-openshift-gitops
+
+toc::[]
+
+With Red Hat OpenShift GitOps, you can define OpenShift cluster configurations such as OpenShift Web Console customizations as well as namespaces by using YAML files and storing them in a Git repository.
+
+This section describes steps to create link:https://github.com/siamaksade/openshift-gitops-getting-started/tree/main/cluster[sample cluster configurations] which add a link to _Red Hat Developer Blog_ in the Application Launcher and  define a namespace `spring-petclinic` on the cluster.
```

Comment:
```suggestion
You can create link:https://github.com/siamaksade/openshift-gitops-getting-started/tree/main/cluster[sample cluster configurations] that add a link to _Red Hat Developer Blog_ in the Application Launcher and define a namespace `spring-petclinic` on the cluster.
```

Reply:
IBMSG: Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes".
Instead, focus on the purpose or value of the content, and include words that users are likely to search
for. For example, in a short description in an online help topic, provide users with enough information to
determine whether the topic meets their needs. You can use self-referential text in learning modules
and tutorials.

Reply:
IBMSG: Use "that", without a comma, to introduce a restrictive clause. Use "which", preceded by a comma, to introduce a nonrestrictive clause.